enclave_manager/avalon_enclave_manager/kme/kme_enclave_manager.py

```diff
@@ -187,7 +211,7 @@ def PreProcessWorkOrder(self, **params):
         encryption_key = params["encryption_key"]
 
         # @TODO : Trusted implementation to be integrated
-        # wo_response = self._execute_work_order(wo_req, encryption_key)
+        # wo_response = self._execute_work_order(wo_request, encryption_key)
```

Comment:
This is commented out, but it should be wo_req, not wo_request

Reply:
@danintel am I missing something? @212, the variable name is wo_request, the same as this one.

Reply:
@rranjan3  You are correct.  Line 212 was hidden so I did not see it--I saw wo_req at line 199.